apps/tenant-management-webapp/src/app/pages/admin/services/access/overview.tsx

```diff
@@ -36,15 +36,15 @@ export const Overview = (): JSX.Element => {
 
   useEffect(() => {
     dispatch(fetchAccess());
-  }, []);
-
-  useEffect(() => {
     return function clean() {
       if (isReady(indicator, userCount)) {
         dispatch(accessReset());
       }
     };
-  }, [indicator]);
+  }, []);
+
+  // eslint-disable-next-line
+  useEffect(() => {}, [indicator]);
```

Comment:
do we need this?

Reply:
Hi @smettu007, thank you for your review. I think we need this - we need to refresh the page when the value is updated. It determines the ready function value, which further determines page clean-up.